packages/components/button/src/button.vue

```diff
@@ -51,6 +87,9 @@ export default defineComponent({
     }
 
     return {
+      elButtonRef,
```

Comment:
**Suggestion:**  Maybe `buttonRef` would be better since we surely not need the prefix for a ref.